packages/marshal/package.json

```
@@ -9,6 +9,7 @@
   },
   "scripts": {
     "build": "exit 0",
+    "prepack": "tsc -p jsconfig.build.json",
```

Can you please add:
```suggestion
    "prepack": "tsc -p jsconfig.build.json",
    "clean": "tsc-emit-clean",
    "postpack": "yarn clean",
```

and take a dependency on `tsc-emit-clean` here?

---

Apparently `yarn tsc --build jsconfig.build.json  --clean` works!